optimize/upgrade/src/main/java/io/camunda/optimize/upgrade/plan/factories/Upgrade313To86PlanFactory.java

```diff
@@ -22,6 +23,13 @@ public UpgradePlan createUpgradePlan(final UpgradeExecutionDependencies dependen
         .fromVersion("3.13")
         .toVersion("8.6.0")
         .addUpgradeStep(new DeleteIndexIfExistsStep("onboarding-state", 2))
+        .addUpgradeStep(new DeleteIndexIfExistsStep("event", 4))
+        .addUpgradeStep(new DeleteIndexTemplateIfExistsStep("event", 4))
+        .addUpgradeStep(new DeleteIndexIfExistsStep("event-process-definition", 5))
+        .addUpgradeStep(new DeleteIndexIfExistsStep("event-process-mapping", 4))
+        .addUpgradeStep(new DeleteIndexIfExistsStep("event-process-publish-state", 4))
+        .addUpgradeStep(new DeleteIndexIfExistsStep("event-sequence-count-external", 4))
+        .addUpgradeStep(new DeleteIndexIfExistsStep("event-trace-state-external", 2))
```

Comment:
are any of these other indices templated?

Reply:
I added a template removal step for the one that was templated, I also spun up optimize (c7 with events disabled) and had a look at a random saas cluster to double check the dynamic CamundaActivityEventIndex doesnt create one if events are disabled, wasnt sure on that one as it is templated but also not created on startup. Would always appreciate a double check though 💯

Reply:
Could you test it on a SaaS cluster with:
```
GET /_template/*
```
From here you could determine what needs deleting explicitly

Reply:
Thats what I meant with `had a look at a random saas cluster`

Reply:
Ok had a look through the code a little deeper, and agree that the template only gets created here on index creation, so should not exist in any Optimize instance that has never had Camunda Activity Events enabled (and imported)